src/base/projects/Issue.svelte

```diff
@@ -136,28 +133,20 @@
   </header>
   <main>
     <div class="comments">
-      <Comment comment={issue.comment} {getImage} {wallet} />
       {#each issue.discussion as comment}
         <Comment {comment} {getImage} {wallet} />
-        {#if comment.replies}
-          <div class="replies">
-            {#each comment.replies as reply}
-              <Comment comment={reply} {getImage} {wallet} />
-            {/each}
-          </div>
-        {/if}
```

Comment:
Why do we remove this?

Reply:
we don't have `.replies` anymore, we `.replyTo`, regardless, we can add them back in once they are working, otherwise `npm run check` throws.